packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -213,7 +213,7 @@ class DevToolsButton extends StatelessWidget {
         tooltip: tooltip,
         tooltipPadding: tooltipPadding,
         child: ElevatedButton(
-          onPressed: _onPressed,
+          onPressed: onPressed,
```

Comment:
this is a logic change as is. These should all use `onPressedHandler`, not `onPressed`